backend.tf

```
@@ -0,0 +1,10 @@
+terraform {
+  backend "s3" {
+    bucket         = "cisa-cool-terraform-state"
```

I'd like to see this skeleton follow the path laid out by PR's such as https://github.com/cisagov/cool-accounts/pull/161 and [all of these other PRs](https://github.com/search?q=org%3Acisagov+%22Support+new+COOL+environment+structure%22&type=pullrequests):
```suggestion
    # Use a partial configuration to avoid hardcoding the bucket name. This
    # allows the bucket name to be set on a per-environment basis via the
    # -backend-config command line option or other methods.  For details, see:
    # https://developer.hashicorp.com/terraform/language/backend#partial-configuration
    bucket         = ""
```

Note that the README should also be updated to include the instructions on how to configure the backend.  Unfortunately, I did not think to put those updated instructions in every PR linked above, but you can refer to https://github.com/cisagov/skeleton-packer/pull/393 for a good example to copy.

---

Please see [cb0e8f2](https://github.com/cisagov/skeleton-tf-root-module/pull/39/commits/cb0e8f22449d33c9617d71f537b5cb911623ee41) and let me know what you think.

---

Looks great- all that I think is missing is a commit like https://github.com/cisagov/publish-egress-ip-terraform/pull/49/commits/60b2c5d1e88407ddfa21c951c4bcaefdef93eed6 to add `*.tfconfig` to the `.gitignore` file.

---

Ah, right, forgot that part. Please see [98e0e68](https://github.com/cisagov/skeleton-tf-root-module/pull/39/commits/98e0e685f5a8f64a228bd5d88bc2130f06aa8528).

---

Perfect, thanks!